### PR TITLE
Fix fileDownloadInfo fetching entirely the file when there is no cred…

### PR DESCRIPTION
**Pull request by Kakulukian**
…entials param

### CODE REVIEW

**Review by coyotte508 — APPROVED** [review]
Thank you

File: packages/hub/src/lib/file-download-info.ts [hunk 1]
```diff
@@ -48,12 +48,14 @@ export async function fileDownloadInfo(params: {
...
-					Authorization: `Bearer ${params.credentials.accessToken}`,
-					Range: "bytes=0-0",
-			  }
-			: {},
+		headers: {
+			...(params.credentials
+				? {
+						Authorization: `Bearer ${params.credentials.accessToken}`,
+				  }
+				: undefined),
```
Comment (coyotte508) [inline packages/hub/src/lib/file-download-info.ts:56]:
```suggestion
			...(params.credentials && {
						Authorization: `Bearer ${params.credentials.accessToken}`,
				  }),
```

In this codebase you can do this if you want :p

Comment (Kakulukian) [reply packages/hub/src/lib/file-download-info.ts:56]: I like that :D

**Review by mishig25 — APPROVED** [review]
lgtm !

**Comment by Kakulukian** [comment]
The tests are failing because we tell to retrieve one byte so Content-length will return 1 (follow the standard) so we may need to use head to get the file size

**Review by coyotte508 — APPROVED** [review]
nice!

**Pull request merged**